app/config.go

```diff
@@ -18,11 +18,12 @@ import (
 
 // CothorityConfig is the configuration structure of the cothority daemon.
 type CothorityConfig struct {
-	Suite       string
-	Public      string
-	Private     string
-	Address     network.Address
-	Description string
+	Suite         string
+	Public        string
+	Private       string
+	Address       network.Address
+	ListenAddress network.Address
```

Comment:
I'm surprised to see a onet/network.Address here. I was expecting this to either be a string, or a net.IP (i.e. already parsed, perhaps via some magic in the toml parser).

Now, it is true that onet/network.Address is in fact just a string, so it's no big deal. But having "tcp://$ip:$port" here, as the documentation implies, doesn't make sense; the syntax of ListenAddress is one single IP address (v4 or v6).